islands/FoldItemsIsland.tsx

```diff
@@ -20,6 +20,27 @@ export interface FoldItem {
    * @default Lorem
    */
   text?: TextArea;
+
+  cta?: {
+    /**
+     * @title Texto CTA
+     * @description Texto do CTA vinculado ao item (mesmo do atributo title).
+     * @default Lorem
+     */
+    textCTA?: TextArea;
+    /**
+     * @title Link CTA
+     * @description Link do CTA vinculado ao item.
+     * @default /#
+     */
+    linkCTA?: TextArea;
```

Comment:
@renan-s-coelho Como já está dentro de `cta`, poderia ser simplificado para `link` ou `href`:
```suggestion
    link?: string;
```